lighthouse-cli/chrome-finder.ts

```diff
@@ -75,14 +75,53 @@ export function darwin() {
 }
 
 export function linux() {
+  const installations: Array<string> = [];
+  const suffixes = [
```

Comment:
These guys use `which`. Any feelings on using that instead?
https://github.com/davidtheclark/locate-chrome/blob/master/index.js#L58